kura/org.eclipse.kura.linux.net/src/main/java/org/eclipse/kura/linux/net/dhcp/server/DnsmasqTool.java

```diff
@@ -50,8 +55,7 @@ public DnsmasqTool(CommandExecutorService service) {
 
     @Override
     public boolean isRunning(String interfaceName) throws KuraProcessExecutionErrorException {
-        CommandStatus status = this.executorService.execute(
-                new Command(new String[] { "systemctl", "is-active", "--quiet", DhcpServerTool.DNSMASQ.getValue() }));
+        CommandStatus status = this.executorService.execute(IS_ACTIVE_COMMAND);
 
         boolean isRunning;
         try {
```

Comment:
Could you please catch only the relevant exceptions in the catch branch (NoSuchAlgorithmException, IOException)? Moreover, the message in the KuraProcessExecutionErrorException is wrong.